.travis.yml

```diff
@@ -3,20 +3,31 @@ rvm:
   - 2.0.0
   - 2.1
   - 2.2
+  - 2.3
+  - 2.4
+  - 2.5
+  - 2.6
   # optional
   - ruby-head
   - jruby-19mode
   - jruby-head
   - rbx-2
 
+addons:
+  hosts:
+    - ldap.example.org # needed for TLS verification
+
+services:
+  - docker
+
 env:
   - INTEGRATION=openldap
 
 before_install:
   - gem update bundler
 
 install:
-  - if [ "$INTEGRATION" = "openldap" ]; then sudo script/install-openldap; fi
+  - docker run --hostname ldap.example.org --env LDAP_TLS_VERIFY_CLIENT=try -p 389:389 -p 636:636 -v $(pwd)/test/fixtures/seed.ldif:/container/service/slapd/assets/config/bootstrap/ldif/50-bootstrap.ldif -v $(pwd)/test/fixtures/openldap/retcode.ldif:/container/service/slapd/assets/config/bootstrap/ldif/06-retcodes.ldif --name openldap --detach osixia/openldap:1.3.0 --copy-service --loglevel debug
```

Comment:
What do you think of breaking the command into multiline lines? I think it reads easier.

We can use either the folded style `>` or the literal style `|` with backslash `\` to break the line.
I have this bookmarked for YAML syntax: [multiline-info](https://yaml-multiline.info/) and [SO](https://stackoverflow.com/a/21699210)

Reply:
Yeah I thought about it and punted because I didn't know if backslash was necessary. Will give it a try!

Reply:
Got it in the third attempt 😅 
https://github.com/ruby-ldap/ruby-net-ldap/pull/333/commits/cf4e9bd5a1af3f1982974ef408043cd0f9664234

WDYT?